localization/english/religion/wc_religion_l_english.yml

```diff
@@ -660,3 +660,27 @@
  masonry_adherent:0 "Mason"
  masonry_adherent_plural:0 "Masons"
  masonry_desc:0 "Arose among the guilds of masons, this branch of the Holy Light opposes itself to the Holy Light Church. They reject dogmas unpopular among the common folk such as the unshakable moral authority of the priests."
+ 
+ avian_group:0 "Avianism"
+ avian_group_adj:0 "Avian"
+ avian_group_adherent:0 "Avian"
+ avian_group_adherent_plural:0 "Avians"
+ avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and the greatest king Terokk they worship as their fourth god."
+ 
+ anzuism:0 "Anzuism"
+ anzuism_adj:0 "Anzuic"
+ anzuism_adherent:0 "Anzuist"
+ anzuism_adherent_plural:0 "Anzuists"
+ anzuism_desc:0 "A sect of $avian_group_adherent_plural$ who worship Anzu, the Raven God. As opposed to Rukhmar loving the sun and sky, Anzu prefer the cool of the shade and the peace of the twilight hours where he can be alone in quiet contemplation. He is ancient enemies with Sethe, who has cursed him long ago to be broken and flightless as well as many Anzu's arakkoa followers."
+ 
+ rukhmarism:0 "Rukhmarism"
+ rukhmarism_adj:0 "Rukhmaric"
+ rukhmarism_adherent:0 "Rukhmarist"
+ rukhmarism_adherent_plural:0 "Rukhmarists"
+ rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After another $avian_group_adherent$ god, Anzu, faded into darkness, she created the arakkoa to represent her grace and his cunning. As the mother of the arakkoa, Rukhmar said to be their main and traditional diety. Those who venerate her are believed to reunited with the mother in the skies."
```

Comment:
```suggestion
 rukhmarism_desc:0 "A branch of $avian_group_adherent_plural$ who worship Rukhmar, the Sun Goddess. After another $avian_group_adherent$ god, Anzu, faded into darkness, Rukhmar created the arakkoa to represent a fusion her grace and Anzu's cunning. As the mother of the arakkoa, Rukhmar is considered by most to be their main and most traditional diety. Those who venerate her are believed to reunite with the mother in the skies after their passing."
```

Reply:
Merged with 14f2be06dd0f615d11951f673f6ea7aaa20b3647 in abfc725060e254b4959f510535ef1421e0f22dba.